### PR TITLE
Psp 7984 node 20 update

### PR DESCRIPTION
**Pull request by devinleighsmith**

### CODE REVIEW

File: source/frontend/public/mockServiceWorker.js
Comment (FuriousLlama) [inline source/frontend/public/mockServiceWorker.js:1]: this file keeps fliping on my machine. Any chance we have conflicting formatting rules?

Comment (devinleighsmith) [reply source/frontend/public/mockServiceWorker.js:1]: potentially, you mentioned that you stopped using vscode, does that mean you aren't using prettier anymore?

Comment (asanchezr) [reply source/frontend/public/mockServiceWorker.js:1]: this happens to me as well anytime Manuel commits changes. I keep ignoring the changes on my local but we should work with the same set of linting rules

Comment (asanchezr) [reply source/frontend/public/mockServiceWorker.js:1]: We should exclude this file from linting regardless of local setup

**Review by asanchezr — APPROVED** [review]

**Pull request merged**